DEPS

```diff
@@ -112,7 +112,7 @@ hooks = [
     # are run with python3, so this environment mirrors the one in which
     # cryptography will be used. Second, we cannot update pip in vpython3 on at
     # least macOS due to permission issues.
-    'action': ['python3', '-m', 'pip', '-q', '--disable-pip-version-check', 'install', '-U', '-t', 'third_party/cryptography', '--only-binary', 'cryptography', 'cryptography==37.0.4'],
+    'action': ['vpython3', '-m', 'pip', '-q', '--disable-pip-version-check', 'install', '-U', '-t', 'third_party/cryptography', '--only-binary', 'cryptography', 'cryptography==37.0.4'],
```

Comment:
why do you want to use `vpython3` if you install dependencies via pip anyways?

in my mind you should either use vpython3 with wheels listed here `https://chromium.googlesource.com/infra/infra/+/main/infra/tools/dockerbuild/wheels.md` or python3 wheels.

combining these two doesn't seem right. By doing this you're polluting vpython3 default environment declared in `src/.vpython3`.

Reply:
I’m aware of the wheels system, but I couldn’t find a wheel for rich. I added this way because we already do this way for other vpython dependencies in this file.

Having this merged will spare us of pip install demands in CI where we have sensitive security context.

Reply:
Actually you’re right… there’s a rich will. I’m gonna pull that… thanks for that links listing all the wheels.

Reply:
I have investigated this a bit further and we can't use their wheel system:

```
INFO: pip is looking at multiple versions of rich to determine which version is compatible with other requirements. This could take a while.
ERROR: Could not find a version that satisfies the requirement markdown-it-py<3.0.0,>=2.1.0 (from rich) (from versions: none)
ERROR: No matching distribution found for markdown-it-py<3.0.0,>=2.1.0
Traceback (most recent call last):
```

I guess I could open a bug with them, to get their markdown wheel updated.

Reply:
> why do you want to use vpython3 if you install dependencies via pip anyways?

I want to use vpython because virtualenv requires pip reinstalls in CI when moving in and out of some of the security contexts (e.g `withCreds`)

Reply:
> I want to use vpython because virtualenv requires pip reinstalls in CI when moving in and out of some of the security contexts (e.g `withCreds`)

I don't see how this is related. Do you mean vpython somehow cleanups all packages when you go out of `withCreds` scope? Is this some next-level integration between google infra tool and jenkins?

If you really want to use vpython, please consider creating a separate `.vpython3` file in your directory, which I believe will lead to a separate venv being created specifically for this `.vpython3` configuration.

Reply:
Fixed.